### PR TITLE
Added is_finite implementation for tf math

### LINKED ISSUE #10681: is_finite

### PR DESCRIPTION
**Pull request by aajinkya1203**
Closes #10681

Added is_finite in math.py along with test case

### CODE REVIEW

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @aajinkya1203 

Thank you for contributing to ivy😊

Your implementation looks good so far🙂
Would you solve the merge conflicts, after that I'll get this PR merged provided that the tests are passing

Thanks again and happy coding🤞

**Comment by aajinkya1203** [comment]
Hello @hirwa-nshuti
I've fixed the merge conflicts. The test cases also seems to be passing. Please let me know what else I can do.

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py [hunk 1]
```diff
@@ -1712,3 +1712,32 @@ def test_tensorflow_is_nan(
...
+    test_flags,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        rtol=1e-02,
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py:1741]:
```suggestion
```

**Review by hirwa-nshuti — APPROVED** [review]
LGTM!

Thanks for your contribution😊

**Pull request merged**